examples/Reranking_search_results_with_cross-encoders.ipynb

```diff
@@ -0,0 +1,805 @@
+       " 'article_url': 'http://arxiv.org/abs/2206.07023v2',\n",
+       " 'pdf_url': 'http://arxiv.org/pdf/2206.07023v2'}"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "result_list[0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "7e6abb5b",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "1: SBERT studies Meaning Representations: Decomposing Sentence Embeddings into Explainable Semantic Features\n",
+      "2: Are Classes Clusters?\n",
+      "3: Semantic Composition in Visually Grounded Language Models\n",
+      "4: Evaluating the Construct Validity of Text Embeddings with Application to Survey Questions\n",
+      "5: Learning Probabilistic Sentence Representations from Paraphrases\n",
+      "6: Exploiting Twitter as Source of Large Corpora of Weakly Similar Pairs for Semantic Sentence Embeddings\n",
+      "7: How to Probe Sentence Embeddings in Low-Resource Languages: On Structural Design Choices for Probing Task Evaluation\n",
+      "8: Clustering and Network Analysis for the Embedding Spaces of Sentences and Sub-Sentences\n",
+      "9: Vec2Sent: Probing Sentence Embeddings with Natural Language Generation\n",
+      "10: Non-Linguistic Supervision for Contrastive Learning of Sentence Embeddings\n",
+      "11: SentPWNet: A Unified Sentence Pair Weighting Network for Task-specific Sentence Embedding\n",
+      "12: Learning Joint Representations of Videos and Sentences with Web Image Search\n",
+      "13: Character-based Neural Networks for Sentence Pair Modeling\n",
+      "14: Train Once, Test Anywhere: Zero-Shot Learning for Text Classification\n",
+      "15: Hierarchical GPT with Congruent Transformers for Multi-Sentence Language Models\n",
+      "16: Sentence-T5: Scalable Sentence Encoders from Pre-trained Text-to-Text Models\n",
+      "17: In Search for Linear Relations in Sentence Embedding Spaces\n",
+      "18: Learning to Borrow -- Relation Representation for Without-Mention Entity-Pairs for Knowledge Graph Completion\n",
+      "19: Efficient and Flexible Topic Modeling using Pretrained Embeddings and Bag of Sentences\n",
+      "20: Relational Sentence Embedding for Flexible Semantic Matching\n"
+     ]
+    }
+   ],
+   "source": [
+    "for i, result in enumerate(result_list):\n",
+    "    print(f\"{i + 1}: {result['title']}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d5727678",
+   "metadata": {},
+   "source": [
+    "## Cross-encoder\n",
+    "\n",
+    "We'll create a cross-encoder using the ```Completions``` endpoint - the key factors to consider here are:\n",
+    "- Make your examples domain-specific - the strength of cross-encoders comes when you fine-tune them to your domain.\n",
```

Comment:
Suggestion: replace 'fine-tune' with 'tailor' or 'customize' or 'adapt' or something, so that people don't think this requires model fine-tuning